### ISSUE TITLE
Counter-Strike - Browser list hacked....

### ISSUE THREAD

**Issue by wilianmaique** [ValveSoftware/halflife#3556]
![image](https://github.com/ValveSoftware/halflife/assets/31930046/226d7d6b-96cf-4458-89ce-01beb67e17a0)

**Comment by Eoin-ONeill-Yokai** [comment]
Yes I've noticed this too. I didn't bother trying to connect when I saw this.

**Comment by KarKarKarKarkarkar** [comment]
Yes, this hackers abuse Fake IP feature from steam networking new update.
